includes/admin/settings.php

```
@@ -24,6 +24,32 @@
+	delete_option( 'knabbel_recent_errors' );
+
+	$redirect_url = add_query_arg(
+		array(
+			'page'                   => 'zw-knabbel-wp-settings',
+			'knabbel_errors_cleared' => '1',
+		),
+		admin_url( 'options-general.php' )
+	);
+
+	wp_safe_redirect( $redirect_url . '#knabbel-recent-errors' );
+	exit;
```

PHPMD (2.15.0)

[error] The function handle_clear_recent_errors() contains an exit expression. (undefined)

(ExitExpression)